pkgs/development/python-modules/pytorch/bin.nix

```diff
@@ -28,7 +29,7 @@ in buildPythonPackage {
 
   format = "wheel";
 
-  disabled = !(isPy37 || isPy38 || isPy39);
+  disabled = !(isPy37 || isPy38 || isPy39 || (isPy310 && stdenv.isLinux));
```

Comment:
Once/before this PR is ready please add a comment accompanying this change explaining why this combo is disabled with a link to the relevant PR/issue.

Reply:
I guess this is not correct since a macosx binary does exist apparently https://github.com/NixOS/nixpkgs/pull/164712/files/2c200e7142f4644bca9c29bc49525cbd53c28450#r830446187

Not sure how I missed it